Fixes #363

fgerling opened the pull request:
Add documentation about building a bigfile from invalid sources.

fgerling committed:
Fixes #363

Add documentation about building a bigfile from invalid sources.

tomschr commented on doc/xml/daps_user_review.xml:
```diff
@@ -112,6 +112,14 @@
   <para> &dapsacr; resolves all XIncludes (replaces them with the referenced content) to create
    the big file. A message is shown where to find the generated output. </para>
 
+  <para> Sometimes error messages points to a line containing an <sgmltag>&lt;xi:include/&gt;</sgmltag>
```
add "... tag" at the end (just for clarification).

tomschr commented on doc/xml/daps_user_review.xml:
```diff
@@ -112,6 +112,14 @@
 
+  <para> Sometimes error messages points to a line containing an <sgmltag>&lt;xi:include/&gt;</sgmltag>
+	     and the real error is hidden somewhere inside the included file.
+		 Using the option <option>--novalid</option> allow you to build and validate a single XML file
```
Maybe move "option" after the `<option>` tag?

tomschr commented on doc/xml/daps_user_review.xml:
```diff
@@ -112,6 +112,14 @@
+  <para> Sometimes error messages points to a line containing an <sgmltag>&lt;xi:include/&gt;</sgmltag>
+	     and the real error is hidden somewhere inside the included file.
+		 Using the option <option>--novalid</option> allow you to build and validate a single XML file
+		 where error messages points to there occurrence in the bigfile.
```
Use "occurance**s**" (plural).

tomschr commented on doc/xml/daps_user_review.xml:
```diff
@@ -112,6 +112,14 @@
+	     and the real error is hidden somewhere inside the included file.
+		 Using the option <option>--novalid</option> allow you to build and validate a single XML file
+		 where error messages points to there occurrence in the bigfile.
+         Use the following command as example: </para>
```
"as" -> "as an"

tomschr reviewed (commented):
Only minor things, see my comments. Otherwise looks good! 👍

fgerling committed:
Update daps_user_review.xml

Changes according @tomschr comments. [1]
[1]: https://github.com/openSUSE/daps/pull/401#pullrequestreview-29192945
